### DISCUSSION TITLE
We have 2 options where to change language

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#52] [Bugs and Reports]
One is on the left menu down, and second one is without the options to choose right down corner. 
<img width="1237" alt="Snímek obrazovky 2024-10-04 v 0 28 53" src="https://github.com/user-attachments/assets/9ed0b5c1-c09b-4b1d-be52-2a8dec130580">

**Comment by JackHipeek** [comment]
Also when i click to another option in menu. like Dynamic staking it set again english like a main language.

**Reply by bf-monster** [reply]
fixed!
